fixtures/app_with_spm_dependencies/LocalSwiftPackage/Package.swift

```diff
@@ -8,9 +8,15 @@ let package = Package(
     defaultLocalization: "en",
     platforms: [.iOS(.v13)],
     products: [.library(name: "Styles", targets: ["Styles"])],
+    dependencies: [
+        .package(path: "../LocalSwiftPackageB")
+    ],
     targets: [
         .target(
             name: "Styles",
+            dependencies: [
+                .product(name: "LibraryA", package: "LocalSwiftPackageB", condition: .when(platforms: [.macOS]))
```

Comment:
Would compilation fail if condition is not parsed? If not, are we really testing it?

Reply:
yes, the source itself includes `import AppKit` when we go to build the iOS app it would fail if the condition is not applied.